lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -14,8 +14,8 @@ def started
         def write(data)
           if data.present?
             json = JSON.parse(data)
-            if @active_engine
-              json["engine_name"] = @active_engine.name
+            if current_engine
```

Comment:
Yeah, I think I will in the feature branch. Leaving it seems more in the spirit of behavior-neutral for this PR.